src/message.rs

```diff
@@ -120,6 +125,38 @@ impl Message {
         Self::from(data)
     }
 
+    /// Returns a raw pointer to the message to be used with ffi calls.
+    pub fn as_message_ptr(&self) -> *const zmq_sys::zmq_msg_t {
+        &self.msg
+    }
+
+    /// Returns a raw mutable pointer to the message to be used with ffi calls.
+    pub fn as_mut_message_ptr(&mut self) -> *mut zmq_sys::zmq_msg_t {
+        &mut self.msg
+    }
+
+    /// Returns the amount of bytes that are stored in this `Message`.
+    pub fn len(&self) -> usize {
+        unsafe { zmq_sys::zmq_msg_size(self.as_message_ptr()) }
+    }
+
+    /// Return `true` is there is at least one byte stored in this `Message`.
```

Comment:
there is *not*?

Reply:
It doesn't have to.

It is a Rust convention to include an `is_empty()` method next to `len()`.

Reply:
It there is at least one bit is_empty will return false :) My point was that the comment is the other way around:

Shouldn't this be:
```
/// Return `true` if there no byte stored in this `Message`.
```

Reply:
Oh right, thanks for pointing it out. I also see there is a typo there.